### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by joshtyf**
Updates:
- use cases
- update features section

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1237,7 +1171,7 @@ MSS:
 
     Use case resumes at step 1.
 
-**Use case: UC117 - Export flashcard data file**
+**Use case: UC116 - Export flashcard data file**
```
Comment (GilbertTan19) [inline docs/DeveloperGuide.md]: I think there is a typo here?

**Review by GilbertTan19 — CHANGES_REQUESTED** [review]
OVerall, LTM! Just a small change needed

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (joshtyf) [reply docs/DeveloperGuide.md]: Oh oops. Okay, will change it
Resolved in 61d8c4d

**Review by GilbertTan19 — APPROVED** [review]
LGTM!

**Pull request merged**